examples/demo/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "scripts": {
     "build": "parcel build index.html",
-    "start": "parcel index.html"
+    "start": "parcel index.html --port 3000 --no-cache"
```

Comment:
I found the code just won't update when developing locally even I was running `yarn watch` under the project root. Not sure what's wrong under the hood so I disable the cache to work around it.